playwright/_impl/_playwright.py

```diff
@@ -40,6 +40,15 @@ def __init__(
             for device in initializer["deviceDescriptors"]
         }
 
+    def __getitem__(self, value: str) -> BrowserType:
```

Comment:
I would inline this exact snippet when generating Playwright (both sync and async), as a single hard-coded blob. That way we don't grow complexity in the generator.